src/classes/InstallScript.cls

```diff
@@ -80,7 +81,12 @@ public without sharing class InstallScript implements InstallHandler {
 
         } catch (Exception ex) {
             // Email error and allow the install to continue.
-            sendError(ex.getMessage());
+            sendError(
+                String.format(
+                    EMAIL_MESSAGE,
+                    new List<String>{ context.organizationId(), ex.getMessage() }
```

Comment:
Would org Id or message ever be null, and if so would this still work?

Reply:
Neither should ever be null but if we passed in null we would get the string formatted with nulls as strings, example:
`System.debug(String.format('test {0} and {1}', new List<String>{ null, null }));`
Results in: 'test null and null'